code/about.html

```diff
@@ -0,0 +1,117 @@
+                <tr>
+                    <th>1981 - 1984</th>
+                    <td>Ausbildung, Handweberin EFZ, BFF Bern</td>
+                </tr>
+                <tr>
+                    <th>1984</th>
+                    <td>Praktikum, Leinenweberei Bern</td>
+                </tr>
+                <tr>
+                    <th>1984 - 1986</th>
+                    <td>Studium, Textildesign FA, Schweizerische Textilfachschule, Wattwil</td>
+                </tr>
+                <tr>
+                    <th>1986 - 1988</th>
+                    <td>Arbeit, Habis Textil AG, Flawil</td>
+                </tr>
+                <tr>
+                    <th>1990 - 1997</th>
+                    <td>Leitung Webatelier, Landwirtschafts- und Bäuerinnenschule, Willisau</td>
+                </tr>
+                <tr>
+                    <th>1991</th>
+                    <td>2. Platz Intern. Design-Preis, Österreich</td>
+                </tr>
+                <tr>
+                    <th>1999 - 2020</th>
+                    <td>Kursleiterin, Zürcher Stalder AG, Kirchberg</td>
+                </tr>
+                <tr>
+                    <th>1999 - 2020</th>
+                    <td>Kursleiterin, Spycher Handwerk Grädel, Huttwil</td>
+                </tr>
+                <tr>
+                    <th>2001 - 2007</th>
+                    <td>Weben in der Beschäftigung, Alters- und Pflegeheim, Lohn-Ammansegg</td>
+                </tr>
+                <tr>
+                    <th>2006 - 2020</th>
+                    <td>Weben und Produktdesign, Stiftung Sonnenbühl, Konolfingen</td>
+                </tr>
+                <tr>
+                    <th>2008 - 2013</th>
+                    <td>Ausbildung, Ausbilderin FA, Migros Klubschule, Bern</td>
+                </tr>
+                <tr>
+                    <th>2008 - 2020</th>
+                    <td>Kursleiterin, Genossenschaft Migros Aare, Wankdorf</td>
+                </tr>
+                <tr>
+                    <th>2009 - 2020</th>
+                    <td>Leitung Webatelier, INFORAMA, Hondrich</td>
+                </tr>
+                <tr>
+                    <th>2009 - 2014</th>
+                    <td>Präsidentin, Verein IG Weben, Berufsverband der WeberInnen in der Schweiz</td>
+                </tr>
+                <tr>
+                    <th>2015 - </th>
+                    <td>Prüfungsexpertin der Gewebegestalterinnen</td>
+                </tr>
+                <tr>
+                    <th>2013 - 2020</th>
+                    <td>Weben und Produktdesign, Huus Regenbogen, Wattenwil</td>
+                </tr>
+                <tr>
+                    <th>2013 - 2015</th>
+                    <td>Studiengang, Farbdesign FA, Schweizerische Textilfachschule, Wattwil</td>
+                </tr>
+                <tr>
+                    <th>2020 - </th>
+                    <td> Olvido, Spiez</td>
+                </tr>
+            </table>
```

Comment:
I'm not 100% sure about this table usage here. I found [this](https://www.thoughtco.com/dont-use-tables-for-layout-3468941) about the topic.